### DISCUSSION TITLE
UI MODIFICATIONS

### DISCUSSION THREAD

**Discussion by perfetimperfect24** [apache/camel-karavan#1333] [Q&A] [answered]
### Ask a question

Is there a way to customize the karavn.space  ui according to business requirements ? If so how should one proceed with
Also can we mock karavan docker ui with karavan .space ui?

### Variant

Web Application

### Container Management (if applicable)

Docker

### Operating System (if applicable)

Linux

### Version

None

**Comment by davsclaus** [comment, marked as answer]
You can fork and do any kind of modifications you want yourself and use that in your own company. Its all front end, css and this kind of stuff. So just dive in the code and do it.
